include/vcpkg/base/message-data.inc.h

```diff
@@ -1099,7 +1099,11 @@ DECLARE_MESSAGE(ErrorInvalidManifestModeOption,
                 (msg::option),
                 "",
                 "The option --{option} is not supported in manifest mode.")
-DECLARE_MESSAGE(ErrorMessage, (), "", "error: ")
+DECLARE_MESSAGE(ErrorMessage,
+                (),
+                "This identifies errors printed on the command line; Visual Studio and other IDEs look for this so "
+                "it's possible that it should remain unlocalized?",
```

Comment:
```suggestion
                "This identifies errors printed on the command line and should remain unlocalized",
```

Translators can't be expected to make this decision for us; they don't know the ins-and-outs of IDEs etc any better than we do (and probably worse).

Reply:
https://github.com/microsoft/vcpkg-tool/pull/1260